### DISCUSSION TITLE
RecursiveSkeletons.gltf is not displayed correctly

### DISCUSSION THREAD

**Discussion by cx20** [google/filament#3765] [Q&A] [answered]
I tried to display the following model in the [latest version](https://github.com/google/filament/actions/runs/722937842) of Filament.
https://github.com/vpenades/glTF-Sample-Models/tree/master/2.0/RecursiveSkeletons

However, unlike the expected result, it seems that a part is missing and displayed.
Is it a matter of usage?

Excepted result:
![image](https://user-images.githubusercontent.com/1833979/113740896-191b8d00-973c-11eb-9eea-7c54a2c87044.png)

[Filament + RecursiveSkeletons.gltf result](https://cx20.github.io/gltf-test/examples/filament/index.html?category=tutorialModels&model=RecursiveSkeletons&scale=0.01&type=glTF):
![image](https://user-images.githubusercontent.com/1833979/113740937-25074f00-973c-11eb-8f5f-736c03a228f6.png)

**Comment by prideout** [comment, marked as answer]
Thanks for the report!  This is due to a bug in `normalizeSkinningWeights`, I'm working on a fix.

**Reply by cx20** [reply]
Thanks for the fix. I have confirmed that this problem has been resolved in the [latest version](https://github.com/google/filament/actions/runs/724783751).

![image](https://user-images.githubusercontent.com/1833979/113896004-7d098880-9804-11eb-8ed6-1fbbaa662060.png)
